extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -93,6 +64,9 @@ void setupClientTransferEncoding(extensions::curl::HTTPClient& client, bool use_
 }  // namespace
 
 void InvokeHTTP::setupMembersFromProperties(const core::ProcessContext& context) {
+  if (!context.getProperty(URL, url_))
+    throw Exception(PROCESS_SCHEDULE_EXCEPTION, "URL property missing or invalid");
```

Comment:
can we use `getRequiredPropertyOrThrow` here?

Reply:
oh I see it was moved, in that case it can stay as is to be easier to review the PR